docs/deploy-with-docker.zh-CN.md

```diff
@@ -0,0 +1,36 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# 使用 Docker 部署
+
+1. 构建镜像
+
+```sh
+$ docker build -t apisix-dashboard:2.0-rc3 .
```

Comment:
Is `2.0-rc3` working? I am not see version in dckerfile.

Reply:
CI passed and working on my side.

Reply:
It's working but I will update it to `docker build -t apisix-dashboard:{$tag} .`